src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -78,7 +77,7 @@ public ResponseEntity fetchItemById(@PathVariable("id") String id) {
      * 400: user not found
      *
      * @param id
-     * @return
+     * @return ResponseEntity(HttpStatus.OK)
```

Comment:
This, as well as the 3 return statements above were just small edits made to the JavaDoc to declare what you were returning. This wasn't a problem anywhere else. 

_note - ignore the spacing edit made in line 67, probably a mistake made on my part_